cli/commands/authenticate.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const authenticationTasks = require('../../lib/authentication-tasks');
+const config = require('../../config');
+const prompt = require('prompt');
+const logger = config.logger;
+
+module.exports = function(program) {
+	program.command('authenticate')
+		.description('Obtain AWS session credentials using a user profile from  ~/.aws/credentials and saves them to ' +
```

Comment:
`Obtain` ... `saves`...    I'd use `Obtains` for grammatical consistency.  :)

Reply:
Also, it's not entirely clear that the point isn't just to copy secret keys from `~/.aws/credentials` to `./.env.session`, it's also do to MFA authentication.